### PR TITLE
Added lambda funcation and configurations for checking bucket size

### LINKED ISSUE #2986: AWS S3: Tool to get the size of our folders
The tool (can use either AWS CLI or API) to compute the size of the folders. UI shows 12.3 TB:
image

Overall beam-outputs:
image

### PR DESCRIPTION
**Pull request by rajnikantsh**
Resolves #2986 

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3026)

### CODE REVIEW

File: aws/src/main/python/s3BucketHandler/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+initscript = (('''#cloud-config
+runcmd:
+  - echo "-------------------checking bucket size----------------------"
+  - sudo dpkg --configure -a
+  - sudo dpkg --remove --force-remove-reinstreq unattended-upgrades
+  - sudo apt-get install unattended-upgrades
+  - sudo dpkg --configure -a
+  - sudo apt update
+  - result=$(aws s3 --region=$REGION ls s3://$BUCKET/$FOLDER --recursive | awk 'BEGIN {total=0}{total+=$3}END{print total/1024/1024" MB"}')
+  - echo "invoke cloudwatch for updating matric..."
```
Comment (JustinPihony) [inline aws/src/main/python/s3BucketHandler/lambda_function.py:15]: nit: spelling mistake

**Review by JustinPihony — APPROVED** [review]
LGTM

File: aws/src/main/python/s3BucketHandler/lambda_function.py (hunk 1 above)
Comment (rajnikantsh) [reply aws/src/main/python/s3BucketHandler/lambda_function.py:15]: Fixed

**Comment by JustinPihony** [comment]
Test!

**Pull request merged**